styles.css

```diff
@@ -293,13 +293,27 @@ article{
 	font-weight:700;
 }
 
+.clearfix {
+	display: block;
+	height: 1%;
+}
+
+.clearfix:after {
+	content: ".";
+	display: block;
```

Comment:
do you have to repeat display block if the parent already has it?

Reply:
Resolved.